canopeum_backend/canopeum_backend/models.py

```diff
@@ -1,16 +1,22 @@
+import re
 from datetime import datetime, timedelta
 from typing import TYPE_CHECKING, ClassVar
 
+import googlemaps
 import pytz
 from django.contrib.auth.models import AbstractUser
 from django.db import models
 from rest_framework.request import Request as drf_Request
 
+from .settings import GOOGLE_API_KEY
+
 # Pyright won't be able to infer all types here, see:
 # https://github.com/typeddjango/django-stubs/issues/579
 # https://github.com/typeddjango/django-stubs/issues/1264
 # For now we have to rely on the mypy plugin
 
+gmaps = googlemaps.Client(key=GOOGLE_API_KEY) if GOOGLE_API_KEY != None else None
```

Comment:
Probably a good idea to handle empty strings too

```suggestion
gmaps = googlemaps.Client(key=GOOGLE_API_KEY) if GOOGLE_API_KEY else None
```